.changeset/tasty-trees-hunt.md

```diff
@@ -0,0 +1,6 @@
+---
+'docs': patch
+'@directus/sdk': major
+---
+
+Fixed directus_extensions endpoint not categorized as a system collection
```

Comment:
Could we summarize this a bit more accurate and reflect the actual changes?
Since it's a breaking change it will receive additional attention in the release notes.

> * re-categorized `directus_extensions` as a system collection
>   * added support for custom fields in the response for this collection
> * exported the missing `updateExtensions` composable
> * documented the `version` property.